CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewTopComponent.java

```diff
@@ -159,9 +166,23 @@ public void newPluginReport(final PluginReport pluginReport) {
         final Graph activeGraph = GraphManager.getDefault().getActiveGraph();
 
         // update the graph report if the new plugin report isn't a low level plugin (which aren't useful as notes)
-        if (activeGraph != null && pluginReport.getGraphReport().getGraphId().equals(activeGraph.getId()) 
-                && !pluginReport.hasLowLevelTag()) {            
-            notesViewPane.setGraphReport(activeGraph, notesViewController);
+        if (activeGraph != null && pluginReport.getGraphReport().getGraphId().equals(activeGraph.getId())
+                && !pluginReport.hasLowLevelTag()) {
+            notesViewPane.setGraphReport();
```

Comment:
relating to the above comment, by passing the active graph here, you can ensure that the active graph being passed is the same graph that was just checked (so it would pass `setGraphReport(activeGraph)` rather than `setGraphReport(GraphManager.getDefault().getActiveGraph())`